wpmautic.php

```diff
@@ -145,19 +150,40 @@ function wpmautic_inject_script() {
 	if ( empty( $base_url ) ) {
 		return;
 	}
-
-	$attrs = wpmautic_get_tracking_attributes();
-
 	?><script type="text/javascript">
 	(function(w,d,t,u,n,a,m){w['MauticTrackingObject']=n;
 		w[n]=w[n]||function(){(w[n].q=w[n].q||[]).push(arguments)},a=d.createElement(t),
 		m=d.getElementsByTagName(t)[0];a.async=1;a.src=u;m.parentNode.insertBefore(a,m)
 	})(window,document,'script','<?php echo esc_url( $base_url ); ?>','mt');
-
-	mt('send', 'pageview'<?php echo count( $attrs ) > 0 ? ', ' . wp_json_encode( $attrs ) : ''; ?>);
 </script>
 	<?php
 }
+/**
+ * Add the mt('send', 'pageview') script with optional tracking attributes
+ */
+function wpmautic_inject_send_script() {
```

Comment:
I think this method can be merged in the `wpmautic_inject_script` one. We can still create the JavaScript function but it'll just create one `<script>` tag in the rendered page.